tinyagentos/routes/todo.py

```
@@ -0,0 +1,269 @@
+    request: Request,
+    user: CurrentUser = Depends(current_user),
+):
+    store = _get_store(request)
+    return await store.create_list(user.user_id, body.title)
+
+
+@router.get("/api/todo/{list_id}")
+async def get_list(
+    list_id: str,
+    request: Request,
+    user: CurrentUser = Depends(current_user),
```

Ruff (0.15.21)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)